### ISSUE TITLE
HTML character entities in package list

### ISSUE THREAD

**Issue by bjarnef** [umbraco/OurUmbraco#624]
In the package list e.g. here https://our.umbraco.com/packages/ many of the descriptions show HTML character entities like `&nbsp;`

![image](https://user-images.githubusercontent.com/2919859/93175666-479a3200-f730-11ea-8276-044cfc9be448.png)

**Comment by nul800sebastiaan** [comment]
Yes and I am extremely wary of doing html decoding here for security reasons. The only thing we can probably safely do is string replace on `&nbsp;` and `&amp;` to at least make those look nicer. But since this is running directly inside the backoffice of Umbraco we need to be super careful that someone doesn't end up adding a malicious `<script>` tag for example.

**Comment by nul800sebastiaan** [comment]
To be fair, there doesn't need to be any `&nbsp;` in these descriptions, so we should replace that with proper spaces anyway.

**Comment by bjarnef** [comment]
I also noticed the character entity `&beta;` = β

![image](https://user-images.githubusercontent.com/2919859/93179753-6996b300-f736-11ea-907b-15a915b9386f.png)

Regarding the old CropUp package I noticed a broken image:
https://our.umbraco.com/packages/website-utilities/eksponent-cropup/

We don't the old ImageGen plugin on Our anymore do we?

```
<img src="../../../umbraco/imagegen.ashx?image=/media/wiki/45370/634702677134890000_LogoUmbraco.png&amp;path=true&amp;width=100&amp;height=100">
```
